simglucose/analysis/risk.py

```diff
@@ -1,17 +1,30 @@
 import numpy as np
-import warnings
-
 
 def risk_index(BG, horizon):
     # BG is in mg/dL
-    # horizon in samples
-    with warnings.catch_warnings():
-        warnings.simplefilter('ignore')
-        BG_to_compute = BG[-horizon:]
-        fBG = 1.509 * (np.log(BG_to_compute)**1.084 - 5.381)
-        rl = 10 * fBG[fBG < 0]**2
-        rh = 10 * fBG[fBG > 0]**2
-        LBGI = np.nan_to_num(np.mean(rl))
-        HBGI = np.nan_to_num(np.mean(rh))
-        RI = LBGI + HBGI
+    BG_to_compute = BG[-horizon:]
+    risks =[risk(r) for r in BG_to_compute]
+    LBGI = np.mean([r[0] for r in risks])
+    HBGI = np.mean([r[1] for r in risks])
+    RI = np.mean([r[2] for r in risks])
+
     return (LBGI, HBGI, RI)
+
+def risk(BG):
+    MIN_BG = 20.0
+    MAX_BG = 600.0
+    if BG <= MIN_BG: 
+        return (100.0, 0.0, 100.0)
```

Comment:
It is not intuitive based on the formula below that why the risk index is saturated at 100 at MIN_BG and MAX_BG. Can you add some comments explaining this?

Reply:
(Sorry, I was using my other github account instead of the jxx123 account.)

Reply:
Those are just the values to which the risk formula was fit. In other words they aimed to make the risk maximum when it was either 20 or 600.
See this paper: https://diabetesjournals.org/care/article/20/11/1655/21162/Symmetrization-of-the-Blood-Glucose-Measurement

In the appendix they derive risk as follows. Note that the units are different mmol/l - but in mg/dl these limits are 20 and 600:
![Screenshot_20231003_110426](https://github.com/jxx123/simglucose/assets/140710/418d3fce-0ca2-4412-afdd-ab247d6cb826)

Reply:
Thanks for the explanation. Can you add a brief comment in the code about this saturation just for code readability? Thanks!

Reply:
Echo the latter comment, I will approve your change once this point is addressed.

Reply:
I've added that to the comment, not sure if you saw.